### DISCUSSION TITLE
Dockerize the setup of FrontOffice for the prod

### DISCUSSION THREAD

**Discussion by bwmravec** [monarc-project/MonarcAppFO#509] [General]
**Describe the solution you'd like**
I'm wondering if there is a plan to dockerize the setup of MonarcAppFO with automated builds through github?

**Comment by ruslanbaidan** [comment]
There is definitely a plan to implement this.

For the moment there are artefacts created for each [release](https://github.com/monarc-project/MonarcAppFO/releases) that contain all the application components built together.
And some [discussions](https://github.com/monarc-project/MonarcAppFO/discussions/454) before.
What is basically needed to use `php:8.1-apache` image, download and unarchive the pre-built release artefact and configure mysql container.
We will look at this later.

For the moment there are possibilities:
- vagrant usage,
- VirtualBox with imported VM,
- manual installation or ansible usage for this.
